File: archives/20190519python/0058. Length of Last Word.py
class Solution:
    def lengthOfLastWord(self, s: str) -> int:
        r=len(s)-1
        while r>=0 and s[r]==" ":
            r-=1
        if r<0:
            return 0
        l=r
        while l>=0 and s[l]!=" ":
            l-=1
        return r-l
        

File: archives/C#/0058. Length of Last Word.cs
public class Solution {
    public int LengthOfLastWord(string s) {
        string allchar="qwertyuiopasdfghjklzxcvbnmQWERTYUIOPASDFGHJKLZXCVBNM";
        int lenS=s.Length-1;
        while (lenS>=0 && s[lenS]==' '){
            lenS-=1;
        }
        int rep=0;
        for(int i=lenS;i>=0;--i){
            if(allchar.Contains(s[i])){
                rep+=1;
            }
            else{
                break;
            }
        }
        return rep;
    }
}
